src/components/OverLimitAlert/AlertContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { PropsWithChildren } from 'react';
+import { Alert, useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+
+export const AlertContainer = ({ children, title }: PropsWithChildren<{ title: string }>) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Alert severity="warning" title={title} className={styles.container}>
+      {children}
+    </Alert>
+  );
+};
+
+const getStyles = () => ({
+  container: css({
+    maxWidth: `1648px`,
```

Comment:
Nitpick, but Is there any reason for this particular value? Shouldn't we use a theme value like `theme.breakpoints.values.xxl`?

Reply:
Yeah, but it's not a very good one 😅 It happens to be the exact width of 4 check group cards and their spacing.

This was a lazy stop gap whilst I was waiting for the other PR you have for redesigning the check group cards.